src/components/DishGroup/DishGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+import { DishGroup as DishGroupProps } from '../../types';
```

Comment:
I'd like to make this component reusable for other types of groups, that's why using particular props from entities is not a good idea. 

you can create types for this component and define that `items` property should have an array of objects. and all of the properties of those objects should be strings.